src/components/mediaManager/AposMediaManager.vue

```diff
@@ -0,0 +1,257 @@
+<!-- 
+  AposMediaManager will be in charge of all media-related state logic
+  this includes doing the selecting and deselecting of items, deciding the editor/selection view,
+  emitting batch action events, etc. All sub views will recieve `media` as a prop
+-->
+
+<template>
+  <AposModal :modal="modal">
+    <template #primaryControls>
+      <AposButton type="default" label="Finished" />
+    </template>
+    <template #leftRail v-if="!!media.length">
+      <AposModalRail>
+        <AposTagList title="Filter by Tag" :tags="tagList" />
+      </AposModalRail>
+    </template>
+    <template #main>
+      <AposModalBody>
+        <template #bodyHeader v-if="!!media.length">
+          <AposMediaManagerToolbar 
+            :selected="selected" :media="myMedia" 
+            v-on:selectClick="selectClick"
+            v-on:trashClick="trashClick"
+            v-on:search="search"
+          />
+        </template>
+        <template #bodyMain>
+          <AposMediaManagerDisplay 
+            v-if="!!myMedia.length" :media="myMedia" ref="display"
+            v-on:edit="updateEditing"
+            v-on:select="select"
+            v-on:selectSeries="selectSeries"
+            v-on:selectAnother="selectAnother"
+          />
+          <div v-else class="apos-media-manager__empty">
+            <AposEmptyState :emptyState="emptyDisplay" />
+          </div>
+        </template>
+      </AposModalBody>
+    </template>
+    <template #rightRail v-if="!!media.length">
+      <AposModalRail type="right">
+        <div class="apos-media-manager__sidebar" :class="{'apos-media-manager__sidebar--empty' : !selected.length}">
+          <AposMediaManagerEditor 
+            v-show="editing" :media="editing" :selected="selected" 
+            v-on:back="updateEditing(null)" v-on:save="saveMedia"
+          />
+          <AposMediaManagerSelections 
```

Comment:
unclear on this .. i'm seeing any time you have only 1 selected or explicitly hit an Edit button you get the editor, otherwise you get the selection template

Reply:
adding additional selections via checkbox now behave as expected when toggling the editor/selection view